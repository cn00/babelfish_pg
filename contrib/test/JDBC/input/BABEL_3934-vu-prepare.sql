CREATE PROCEDURE babel_3934_1 AS BEGIN DECLARE @A AS int DECLARE @C int SELECT TOP(1) @A = A, @B = B FROM tbl END
GO

CREATE PROCEDURE babel_3934_2 AS BEGIN RETURN 1 END
GO