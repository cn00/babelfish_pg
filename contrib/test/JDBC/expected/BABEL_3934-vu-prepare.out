CREATE PROCEDURE babel_3934_1 AS BEGIN DECLARE @A AS int DECLARE @C int SELECT TOP(1) @A = A, @B = B FROM tbl END
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: "@B" is not a known variable)~~


CREATE PROCEDURE babel_3934_2 AS BEGIN RETURN 1 END
GO
